aws/rhel-10.0-nightly-aarch64/config.json

```diff
@@ -0,0 +1,5 @@
+{
+    "user": "ec2-user",
+    "runnerArch": "aarch64",
+    "prepareScript": "sudo rm -f /etc/yum.repos.d/*.repo; sudo tee /etc/yum.repos.d/rhel9internal.repo <<EOT\n[RHEL-9-NIGHTLY-BaseOS]\nname=baseos\nbaseurl=http://download.devel.redhat.com/rhel-9/nightly/RHEL-9/latest-RHEL-9.5/compose/BaseOS/aarch64/os/\nenabled=1\ngpgcheck=0\n\n[RHEL-9-NIGHTLY-AppStream]\nname=appstream\nbaseurl=http://download.devel.redhat.com/rhel-9/nightly/RHEL-9/latest-RHEL-9.5/compose/AppStream/aarch64/os/\nenabled=1\ngpgcheck=0\n\n[RHEL-9-NIGHTLY-CRB]\nname=crb\nbaseurl=http://download.devel.redhat.com/rhel-9/nightly/RHEL-9/latest-RHEL-9.5/compose/CRB/aarch64/os/\nenabled=1\ngpgcheck=0\nEOT"
```

Comment:
The urls here should be updated to `https://download.devel.redhat.com/rhel-10/nightly/RHEL-10-Public-Beta/latest-RHEL-10/` I think

Reply:
Aslo I guess the repo file could be updated to `rhel10internal.repo`

Reply:
And the name in the `[]` should be updated as well.